src/docfx.website.themes/default/toc.html.js

```diff
@@ -6,6 +6,11 @@ exports.transform = function (model) {
   return model;
 
   function transformItem(item, level) {
+    // set to null incase mustache looks up
+    item.href = item.href || null;
+    item.tocHref = item.tocHref || null;
```

Comment:
topicHref is not used in js, so may not update?

---
In reply to: [86301821](https://github.com/dotnet/docfx/pull/853#discussion_r86301821) [](ancestors = 86301821)